tests/test_agent.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+
+from reppy.agent import Agent
+
+
+class AgentTest(unittest.TestCase):
+    '''Tests about the Agent.'''
+
+    def test_basic(self):
+        '''Parses a basic example.'''
+        agent = Agent().allow('/').disallow('/foo')
+        self.assertEqual(len(agent.directives), 2)
+
+    def test_checks_allowed(self):
+        '''Answers the allowed question.'''
+        agent = Agent().allow('/path')
+        self.assertTrue(agent.allowed('/path'))
```

Comment:
I think it would be nice to write a helper `self.assert_allowed(agent, path)` and `self.assert_disallowed(agent, path)` to make these tests a bit more readable.